resources/Materials/TestSuite/stdlib/color_management/color3_vec3_cm_test.mtlx

```diff
@@ -1,40 +1,40 @@
 <?xml version="1.0"?>
 <materialx version="1.37" colorspace="lin_rec709">
 
-    <!-- create nodegraph to test cm conversion on image node before feeding to heighttonormal  -->
-    <nodegraph name="height_to_norma_cm">
-        <image name="b_image" type="color3">
-            <parameter name="file" type="filename" value="resources/Images/brass_old_spec.jpg" colorspace="srgb_texture"/>
-            <parameter name="uaddressmode" type="string" value="periodic"/>
-            <parameter name="vaddressmode" type="string" value="periodic"/>
-        </image>
-        <extract name="extract1" type="float">
-            <input name="in" type="color3" nodename="b_image"/>
-            <parameter name="index" type="integer" value="1"/>
-        </extract>
-        <heighttonormal name="impl_heighttonormalmap" type="vector3">
-            <input name="in" type="float" nodename="extract1"/>
-            <parameter name="scale" type="float" value="1.0"/>
-        </heighttonormal>
-        <output name="height_normal_map_output" type="vector3" nodename="impl_heighttonormalmap"/>
-    </nodegraph>
+  <!-- create nodegraph to test cm conversion on image node before feeding to heighttonormal  -->
+  <nodegraph name="height_to_norma_cm">
```

Comment:
minor: probably not worth mentioning as it was wrong before, but should it be height_to_normal_cm?